export/securedrop_export/print/print_dialog.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import gi
+
+gi.require_version("Gtk", "4.0")
+
+from gi.repository import Gtk  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+
+def open_print_dialog(file_to_print):
+    app = PrintDialog(file_to_print)
+    app.run()
+
+
+class PrintDialog(Gtk.Application):
+    def __init__(self, file_to_print):
+        super().__init__(application_id="org.securedrop.PrintDialog")
+        self.file_to_print = file_to_print
+        self.connect("activate", self.on_activate)
+
+    def on_activate(self, app):
```

Comment:
I would find it useful if you could add brief documentation comments explaining when each on_ function is triggered, just since Gtk is still new for most of us.